main.yml

```
@@ -400,6 +400,44 @@ parameters:
           # Pull in the APPUiO managed info labels
           * on(cluster_id) group_left(sales_order) appuio_managed_info{
               cloud_provider=~"%(cloud_provider)s",
+              distribution=~"openshift4",
+              vshn_service_level="%(vshn_service_level)s",
+          }
+
+      appuio_managed_cores:
```

To be in line with `appuio_managed_vcpu` I would name this `appuio_managed_core`:

```suggestion
      appuio_managed_core:
```